### PR TITLE
rpl: init at 1.5.6

### PR DESCRIPTION
**Pull request by teto**
A python program that helps replacing string in files.

###### Motivation for this change
One hell of a program, kinda like "fd". I find it very helpful to replace strings in multiple files. It's packaged in debian too.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -19269,6 +19269,8 @@ in {
     };
   };
 
+  rpl = callPackage ../tools/text/rpl {};
```
Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: If it is not a python package, please move it to `all-packages` instead.

File: pkgs/tools/text/rpl/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  pname = "rpl";
+  version = "1.5.7";
+
+  # Tests not included in pip package.
+  doCheck = false;
+
+  src = fetchFromGitHub {
+    owner="kcoyner";
+    repo="rpl";
+    rev="v${version}";
```
Comment (vyp) [inline pkgs/tools/text/rpl/default.nix]: Really nitpicky so sorry, but spaces around `=` for these?

File: pkgs/top-level/python-packages.nix (hunk 1 above)
Comment (teto) [reply pkgs/top-level/python-packages.nix]: it's a python program but not a library ? still going to all-packages.nix ?

Comment (Mic92) [reply pkgs/top-level/python-packages.nix]: Yes. We need to build python programs only against a single python version. pythonPackages are build against all versions.

File: pkgs/tools/text/rpl/default.nix (hunk 2 above)
Comment (teto) [reply pkgs/tools/text/rpl/default.nix]: I've aligned = with vim-lion, hopes that's not a problem xD

**Comment by teto** [comment]
I pushed a new version with your comments adressed (hopefully).

**Comment by Mic92** [comment]
Thanks!

**Pull request merged**